src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,4 @@
+.title {
```

Comment:
@KovalivNataliia while it is totally acceptable to give classes general name ('Title') in the sub-features and sub-components, it is best practice to organise names corresponding to the names of the component, unless you are re-defining Global CSS classes. I.e. name .header-title would be more suited.